socorro/processor/rules/mozilla.py

```diff
@@ -434,7 +434,7 @@ def __init__(self, schema):
 
         # NOTE(willkg): if the "breadcrumbs" section ever gets moved in the processed
         # crash schema, we'll need to update this
-        self.BREADCRUMBS_SCHEMA = schema["definitions"]["breadcrumbs"]
+        self.breadcrumbs_schema = schema["properties"]["breadcrumbs"]
```

Comment:
Pulling from "definitions" means we're using a schema that hasn't had references resolved. `breadcrumbs` doesn't currently have references, but it might at some point and it seems prudent to change this to pull from "properties" and use a schema where references have been resolved.